exercises/J-functions/exercise.js

```diff
@@ -1,5 +1,5 @@
 function halve(number) {
-  // complete the function here
+	return number / 2;
```

Comment:
This is great Haroon, the only thing I usually advise is anytime you are doing a calculation or comparison etc, wrap the code in parenthesis to make it more clear. This becomes easier to see the benefit when you have multiple things happening in one line as it helps break it down into readable sections.

```javascript
function halve(number) {
    return (number / 2);
}
```

Reply:
same as my reply to https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1/pull/369/files#r972645766

Reply:
Hi Haroon, if you're using a beautifier which alters your code which CYF suggests you use, it's okay. This was more of a suggestion on writing more readable code. You don't actually have to make the change :)